crates/vault-registry/src/types.rs

```diff
@@ -453,10 +619,18 @@ impl<T: Config> RichVault<T> {
         Pallet::<T>::get_backing_collateral(&self.id())
     }
 
+    pub fn get_personal_secure_threshold(&self) -> Result<UnsignedFixedPoint<T>, DispatchError> {
```

Comment:
```suggestion
    pub fn get_custom_secure_threshold(&self) -> Result<UnsignedFixedPoint<T>, DispatchError> {
```

Reply:
On second thought, since this is operating directly on the vault object, how about just using `set_secure_threshold` and `get_secure_threshold` since you return anyway either the global or vault specific threshold here. Naming it `get_custom..` or `get_personal..` implies it would not fetch the global one and return `None` if unset.